website/content/en/preview/upgrade-guide.md

```diff
@@ -105,6 +105,8 @@ Snapshot releases are tagged with the git commit hash prefixed by the Karpenter
 ### Upgrading to v0.29.0+
 * Karpenter has changed the default metrics service port from 8080 to 8000 and the default webhook service port from 443 to 8443. In `v0.28.0`, the Karpenter pod port was changed to 8000, but referenced the service by name, allowing users to scrape the service at port 8080 for metrics. `v0.29.0` aligns the two ports so that service and pod metrics ports are the same. These ports are set by the `controller.metrics.port` and `webhook.port` helm chart values, so if you have previously set these to non-default values, you may need to update your Prometheus scraper to match these new values.
 
+* Karpenter will now reconcile nodes that are drifted due to their Security Groups or their Subnets. If your AWSNodeTemplate's Security Groups differ from the Security Groups used for an instance, Karpenter will consider it drifted. If the Subnet used by an instance is not contained in the allowed list of Subnets for an AWSNodeTemplate, Karpenter will also consider it drifted.
+
 ### Upgrading to v0.28.0+
```

Comment:
Should we shorten the `Warning` in the Upgrade Guide and just make reference to the threat model for the tag-based escalation of privilege?